Gemfile

```diff
@@ -27,6 +27,7 @@ group :development, :test do
   gem 'coveralls', require: false
   gem 'chromedriver-helper'
   gem 'selenium-webdriver'
+  gem 'launchy'
```

Comment:
This one is not really needed. With Selenium, if you want to inspect what the page looks like in the browser, you just insert a `Then stop` step and the execution will halt. At that point you can inspect the elements and run any js code in the console (as long as it does not affect the back-end)